### ISSUE TITLE
[Suggestion] Expose spawn_args

### ISSUE THREAD

**Issue by chinese-soup** [YaLTeR/bxt-launcher#1]
Hello.

Could the spawn_args (or "additional" spawn_args other than the binary and "-steam") be exposed in the GUI so that you can specify command line arguments to the Half-Life process?
`string[] spawn_args = {"./hl_linux", "-steam"};`

This would be helpful for spawning gamemodes like Blue-Shift, OP4, 3rd party etc. and other things that I am pretty sure are obvious to you.

Thanks.

**Comment by YaLTeR** [comment]
Hey, so here are the ideas I have for this launcher:
- I want it to scan the Half-Life folder for mod folders and offer them in a dropdown with human-readable names where possible. I think it can be done by going through the folders, selecting the ones that have `liblist.gam` and taking the name from the `game "Good Name"` lines.
- I also want a CLI for launching stuff, so (modulo bike-shedding) `./bxt-launcher --launch` would just start the game immediately, along with `./bxt-launcher --launch <additional hl args>` for the additional arguments.

Since the second point already allows for full flexibility if needed, the GUI can then be populated only with the actually useful things; I need to actually use the launcher more actively to figure out what these useful things would be (past the obvious mod selection as I outlined). My current opinion is that a full-fledged "additional arguments" GUI entry will serve as a famous "permament temporary solution", so I don't want to add it into the GUI just yet.

This "actually using the launcher" step wasn't going as well as I hoped it would because I currently run HL under the Flatpak Steam to get a consistent TASing environment (yay desyncs caused by different libm.so between Arch and Fedora) and the Flatpak runtimes apparently don't ship with libgtop needed for the launcher and I'm also doing something wrong with d-conf which prevents it from running under there.

To summarize, the current course of action for me would be:
1. add proper mod selection dropdown
2. fix running under Flatpak Steam so I can dogfood the launcher
